components/utilities/direction/direction-settings.jsx

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
```

Comment:
Added this file under `components/utilities/direction/direction-settings.jsx` instead of `components/utilities/private/direction/direction-settings.jsx` since I didn't feel the need for `private` folder. Let me know if you think otherwise.